Lib/typing.py

```diff
@@ -440,8 +442,32 @@ def NoReturn(self, parameters):
       def stop() -> NoReturn:
           raise Exception('no way')
 
-    This type is invalid in other positions, e.g., ``List[NoReturn]``
-    will fail in static type checkers.
+    Static type checkers treat this as a general bottom type,
+    a type with no members. The typing.Never type provides a
+    more explicit name for that concept.
+
+    """
+    raise TypeError(f"{self} is not subscriptable")
+
+@_SpecialForm
+def Never(self, parameters):
+    """Notation for the bottom type, a type that has no members.
```

Comment:
I'd drop "Notation for". This *is* the bottom type, right? In the end, *everything* is a notation for some concept, but we don't normally use that phrase.